ui-components/src/headers/Hero.tsx

```diff
@@ -44,11 +45,14 @@ const Hero = (props: HeroProps) => {
   if (props.centered) {
     classNames = "centered"
   }
+  if (props.secondaryTitle) {
```

Comment:
such a nit, but what do you think about moving this `if` up so that it's just below

```
  if (props.backgroundImage) {
    styles = { backgroundImage: `url(${props.backgroundImage})` }
  }
```

(Just to mimic the way that `secondaryTitle` is right below `subHeader`)

Reply:
updated